### DISCUSSION TITLE
Mi Band

### DISCUSSION THREAD

**Discussion by GLoren1** [Navid200/xDrip#185] [Watches]
Hi.  Is Mi Band 5 and Mi Band 6 are now supported?

I've read https://github.com/NightscoutFoundation/xDrip/issues/1872 , but still can't determine whether it's supported. 

Thanks

**Comment by Navid200** [comment]
May I ask why you posted here instead of in the main repository discussions?
https://github.com/NightscoutFoundation/xDrip/discussions
Is there any guide somewhere that points to these discussions here?  If yes, please provide a link?

**Comment by GLoren1** [comment]
I saw the link posted at https://github.com/NightscoutFoundation/xDrip/issues/1872
Apologies for posting in the wrong place. 

I just re-posted it here under Q&A:
https://github.com/NightscoutFoundation/xDrip/discussions/2003

**Reply by Navid200** [reply]
No, no problem at all.  And thanks for helping me see what I have forgotten to update.
And thanks for posting in the main repository discussions.
